jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -227,8 +202,27 @@ public void waitForLoggingThread() {
   }
 
   @Override
-  public EventHandlers getEventHandlers() {
-    return eventHandlers;
+  public void applyEventHandlers(Containerizer containerizer) {
+    containerizer
+        .addEventHandler(LogEvent.class, this::log)
+        .addEventHandler(
+            TimerEvent.class,
+            new TimerEventHandler(message -> consoleLogger.log(LogEvent.Level.DEBUG, message)))
```

Comment:
This is a minor point, but WDYT of making `ConsoleLogger::log` accept `LogEvent`? That is, `consoleLogger.log(LogEvent.debug(message))`.

Reply:
It doesn't seem much neater to me than `consoleLogger.log(Level.DEBUG, message)`, so I think I'll leave it as is for now so the PR doesn't get too muddy. Maybe in a follow up if we want.